hls/indcpa.cpp

```diff
@@ -207,7 +213,11 @@ void indcpa_keypair_derand(bit8_t pk[KYBER_INDCPA_PUBLICKEYBYTES],
   bit8_t nonce = 0;
   polyvec a[KYBER_K], e, pkpv, skpv;
 
-  memcpy(buf, coins, KYBER_SYMBYTES);
+  //memset(buf, 0, KYBER_SYMBYTES);
+  for (i = 0; i < KYBER_SYMBYTES; i++) {
+    buf[i] = 0;
+  }
+  // memcpy(buf, coins, KYBER_SYMBYTES);
```

Comment:
This seems odd to me, the ref originally set this to randombytes, is 0 safe here? @fangzhonglyu

Reply:
This is a temporart thing. Right now this will guarantee deterministic behavior so we can compare with ref